### PR TITLE
Basic role for consistently creating roles.

### PR DESCRIPTION
**Pull request by e0d**
I've been creating a lot of new roles and thought something like this would be useful.  It creates a role and stubs out some files.  I'd like it to include a header file in all the templates, but it's not working currently.  I think this would help us enforce consistency as we create more and more roles.

Runs something like so.

```
ansible-playbook ansible_role.yml -i local.ini --connection=local --extra-vars "role_name=haproxy"
```

### CODE REVIEW

File: playbooks/roles/ansible-role/templates/header.j2 [hunk 1]
```diff
@@ -0,0 +1,3 @@
+#
+# This file was created by the edX ansible-role ansible role.
+#
```
Comment (feanil) [inline playbooks/roles/ansible-role/templates/header.j2]: Where does this come in to play?  Doesn't seem like this file gets called/put in the new role.

**Comment by e0d** [comment]
Includes now working.

**Comment by feanil** [comment]
cool, looks good, did you need to make it a '.txt' to get it to work?

**Comment by jarv** [comment]
looks good to me, the only addition we might consider is adding a deploy.yml though i guess that won't apply to every single role.

**Pull request merged**